daemons/based/based_callbacks.c

```diff
@@ -1178,6 +1178,28 @@ calculate_section_digest(const char *xpath, xmlNode * xml_obj)
 
 }
 
+// v1 and v2 patch formats
+#define XPATH_CONFIG_CHANGE             \
+    "//" XML_CIB_TAG_CRMCONFIG " | "    \
+    "//" XML_DIFF_CHANGE                \
+    "[contains(@" XML_DIFF_PATH ",'/" XML_CIB_TAG_CRMCONFIG "/')]"
+
+static bool
+has_config_change(xmlNode *diff)
```

Comment:
nitpick: contains_config_change() (I initially read it like "Has the config changed?")

Reply:
"Contains" definitely works well for me. I tossed around ideas like `did_config_change(), `is_config_change()`, etc. First thought was `config_changed()`, but it's used as a lot of variable names.